packages/pool/contracts/StateUtils.sol

```diff
@@ -344,6 +365,26 @@ contract StateUtils is IStateUtils {
             );
     }
 
+    /// @notice Called by a DAO Api3Voting app to update the last vote snapshot
+    /// block number
+    /// @param snapshotBlock Last vote snapshot block number
+    function updateLastVoteSnapshotBlock(uint256 snapshotBlock)
+        external
+        override
+    {
+        bool notAuthorized = true;
+        for (uint256 i = 0; i < votingApps.length; i++)
```

Comment:
I would agree that if we do not want to have more than two voting apps it might be easier to just do a simple require statement f.e.:

`require(votingApps[0] == msg.sender || votingApps[1] == msg.sender )`

Reply:
I agree, just mentioned a similar thing here https://github.com/api3dao/api3-dao/pull/143#discussion_r603296717